commune_app/models.py

```diff
@@ -1,3 +1,44 @@
-# from django.db import models
+from django.db import models
```

Comment:
maybe it will be better to create a "models" folder and put this and all the modules files in it like:
models -
   |_ users.py
   | _ commune.py
...